src/main/java/org/example/homework5/NumberGenerator.java

```diff
@@ -0,0 +1,31 @@
+package org.example.homework5;
+
+import java.util.Random;
+
+public class NumberGenerator {
+
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] num = new int[101];
+        int sum = 0;
+        int counter = 0;
+
+        for (int i = 0; i < num.length; i++) {
+            num[i] = 1000;
```

Comment:
Why 1000. Don't see a logic? 
You actually don't need to fill the array before.
You can delete it and check))